src/api.v2/model/Experiment.js

```diff
@@ -0,0 +1,165 @@
+const _ = require('lodash');
+
+const BasicModel = require('./BasicModel');
+const sqlClient = require('../../sql/sqlClient');
+const { collapseKeyIntoArray } = require('../../sql/helpers');
+
+const { NotFoundError } = require('../../utils/responses');
+
+const getLogger = require('../../utils/getLogger');
+
+const logger = getLogger('[ExperimentModel] - ');
+
+const experimentTable = 'experiment';
+const experimentExecutionTable = 'experiment_execution';
+const userAccessTable = 'user_access';
+const metadataTrackTable = 'metadata_track';
+
```

Comment:
We can move all these files to config to avoid duplicate definition of table names